### DISCUSSION TITLE
Right Sidebar on Fuma's own doc page

### DISCUSSION THREAD

**Discussion by DataHawker** [fuma-nama/fumadocs#756] [General]
I love the retro shape of the right sidebar on the official fumadocs page and how is highlights multiple sections at once. Is there a boiler for this around here?

**Comment by fuma-nama** [comment]
This is a built-in option of TOC, see https://fumadocs.vercel.app/docs/ui/blocks/page#options

Enable `clerk` style, and disable `single` to get the same look as the official docs.

I'm planning to make `single` disabled by default on next major releases, since it's more likely a personal preference.

**Reply by DataHawker** [reply]
Thanks! I totally missed that. Looks great.
